src/app/header/header.component.ts

```diff
@@ -0,0 +1,10 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'header',
```

Comment:
Don't have single word selectors for Angular Components.

They may overlap with actual HTML Elements.

In this case, 
```html
<header>
```
is an actual HTML Element which some people may want to use semantically.. but then your component will rewrite their template and stuff lol